rollup/core/classes/RollupLogger.cls

```diff
@@ -51,6 +57,36 @@ public virtual class RollupLogger extends Rollup implements ILogger {
     // this is a no-op by default; sub-classes can opt in if they need to perform DML
   }
 
+  private RollupLoggerPluginParameter__mdt loggingLevel {
```

Comment:
Not a major issue (especially since this variable is `private`), but it seems a little misleading to call this `loggingLevel` when it's not an instance of `LoggingLevel` enum. `loggingLevelParameter` would be more descriptive, but not a big deal if you want to keep it as-is

Reply:
err yes, case of the misnamed variable (and I confused myself on this while writing it) - good call, I will update this